### DISCUSSION TITLE
libmopac.so.1: undefined reference to `omp_get_max_threads_' libmopac.so.1: undefined reference to `omp_set_num_threads_'

### DISCUSSION THREAD

**Discussion by 123xyl** [openmopac/mopac#100] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/96686347/183250758-0744e683-ce20-451d-a8b9-951bde8c29f5.png)

**Comment by susilehtola** [comment, marked as answer]
This was fixed in
https://github.com/openmopac/mopac/releases/tag/v22.0.4
